app/helpers/bgeigie_imports_helper.rb

```diff
@@ -2,20 +2,20 @@ module BgeigieImportsHelper
 
   def collect_bgeigie_logs(bgeigie_logs)
     bgeigie_logs.collect do |b|
-      point = {
+      {
         :lat => b.latitude, 
         :lng => b.longitude,
         :cpm => b.cpm
       }
     end
   end
 
-  def bgeigie_nav_li(status)
+  def bgeigie_nav_li(status) # rubocop:disable Metrics/AbcSize
     active = if params[:by_status].blank?
       status == :all
     else
       params[:by_status] == status.to_s
-    end
+             end
```

Comment:
I believe that the indentation of `else` will be checked by another rubocop rule. I would like to leave it for now since rubocop doesn't warn (yet).